examples/favorite/src/constants.ts

```diff
@@ -0,0 +1,14 @@
+export const LOCAL_STORAGE_KEY = 'fs-list-favorite';
+
+export const LIST_INSTANCE_ATTRIBUTE = 'fs-list-instance';
+
+export const FAVORITE_TARGET_ATTRIBUTE = 'fs-list-favorite';
+export const FAVORITE_ELEMENT_ATTRIBUTE = 'fs-list-element';
+export const FAVORITE_ADD_BUTTON = 'favorite-add';
+export const FAVORITE_REMOVE_BUTTON = 'favorite-remove';
+export const FAVORITE_TOGGLE_BUTTON = 'favorite-toggle';
+export const LIST_ITEM = 'item';
+
+export const FAVORITE_CLASS = 'is-list-favorite';
+export const FAVORITE_DISABLED_CLASS = 'is-list-favorite-disabled';
+export const FAVORITING_CLASS = 'fs-list-favoriting';
```

Comment:
is there any value in this list that user can change to their preferences?
For example if I want to rename `is-list-favorite` to something else `is-cartitem-favorite`

Reply:
The only Attributes that are required to be fs-list for compatibility are:
- `fs-list-element="list"`
- `fs-list-element="item"`

The rest can use a different namespace.

Note: the constants above are outdated, check my latest update